website/static/js/pages/wiki-edit-page.js

```diff
@@ -167,3 +168,25 @@ $(document).ready(function () {
     // Tooltip
     $('[data-toggle="tooltip"]').tooltip();
 });
+
+var $comments = $('.comments');
+if ($comments.length && window.contextVars.wiki.wikiID !== 'None') {
+    var currentUser = {
```

Comment:
Good catch! I'll need to change the currentUser info I construct from API JSON data as well since the url is under the `urls.profiles` key.